assets/src/hooks/useDetour.ts

```diff
@@ -28,8 +28,12 @@ export const useDetour = (input: UseDetourInput) => {
   // Record snapshots when changed
   useEffect(() => {
     const snapshotSubscription = actorRef.subscribe(() => {
-      const serializedSnapshot = JSON.stringify(actorRef.getPersistedSnapshot())
+      const persistedSnapshot = actorRef.getPersistedSnapshot()
+      const serializedSnapshot = JSON.stringify(persistedSnapshot)
       localStorage.setItem("snapshot", serializedSnapshot)
+      if (persistedSnapshot.context.uuid) {
+        putDetourUpdate(persistedSnapshot.context)
```

Comment:
issue: I think the entire `persistedSnapshot` object needs to be saved? not just the `context`